packages/trace-viewer/src/ui/modelUtil.ts

```diff
@@ -410,10 +410,14 @@ function collectSources(actions: trace.ActionTraceEvent[], errorDescriptors: Err
   return result;
 }
 
-const kRouteMethods = new Set([
+const kRouteMethods = [
   'page.route', 'page.routefromhar', 'page.unroute', 'page.unrouteall',
   'browsercontext.route', 'browsercontext.routefromhar', 'browsercontext.unroute', 'browsercontext.unrouteall',
-]);
+];
 export function isRouteAction(action: ActionTraceEventInContext) {
-  return action.class === 'Route' || kRouteMethods.has(action.apiName.toLowerCase());
+  if (action.class === 'Route')
+    return true;
+  // action.apiName is coming from the client. In e.g. .NET its suffix is Async, so we
```

Comment:
Should we also remove `_` to make python work? Perhaps just list all lang methods in the set, and remove `toLowerCase()`?